liwords-ui/src/gameroom/table.tsx

```diff
@@ -555,6 +555,9 @@ export const Table = React.memo((props: Props) => {
             playerMeta={gameInfo.players}
             tournamentID={gameInfo.tournament_id}
             lexicon={gameInfo.lexicon}
+            handleAcceptRematch={
+              rematchRequest.getRematchFor() !== '' ? handleAcceptRematch : null
```

Comment:
i can see an edge case here where you got a rematch for a separate game in a different tab (although, i may already handle that properly with the connection ID stuff). What if we say `getRematchFor() === gameID`